src/renderers/shared/fiber/isomorphic/ReactPortal.js

```diff
@@ -27,6 +27,7 @@ exports.createPortal = function(
   implementation: any,
   key: ?string = null,
 ): ReactPortal {
+  containerInfo._reactPortalContainer = {};
```

Comment:
This seems unexpected to me. `createPortal` looks like a pure function (like `createElement`) and I wouldn’t expect it to put a property on its argument. Can we do this when the portal mounts for the first time instead? I’m not sure where this code would go though. This is probably not a big deal in grand scheme of things but it does feel odd.

Another concern is it might already exist. So this always allocates an object. I think we should create at most one object.

It would make me feel better about it if this was a boolean rather than an object, and if we only set it in `__DEV__`. Then it’s clear it shouldn’t be relied on, and it‘s only used for warning rather than keeping data or important logic. In this case I don’t mind side effect that much.